src/pages/Signup/Signup.module.css

```diff
@@ -0,0 +1,111 @@
+.products-header {
+  background-image: url("../../assets/images/background/shop-banner.jpg");
+  position: relative;
+  min-height: 60vh;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+
+
+.products-header .banner-text {
+  padding-top: 8rem;
+  width: 100%;
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  align-items: center;
+  color: var(--COLOR-white);
+  font-size: 2rem;
+}
+
+.products-header .banner-products {
+  padding-top: 1rem;
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  gap: 2rem;
+  padding-bottom: 2rem;
+}
+
+.banner-products.wishlist{
+    gap: 0.5rem;
+}
+
+.products-header .banner-products a {
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  gap: 0.5rem;
+  font-weight: 500;
+  font-size: 1rem;
+  text-decoration: none;
+  color: var(--COLOR-footer-text);
+}
+.login-main{
+    display: flex;
+    flex-flow: row;
+    padding: 2rem;
+    gap: 2rem;
+}
+
+.login-main hr{
+    border: none;
+    border-right: 2px solid var(--COLOR-border-color-quantity);
+}
+
+.login-box{
+    padding: 3rem;
+    flex: 1 1 50%;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    gap: 1rem;
+}
+.login-box .login-header{
+    color: var(--COLOR-primary-green);
+    font-weight: 500;
+    font-size: 2rem;
+}
+
+.login-box .login-input{
+    display: flex;
+    flex-flow: column;
+    gap: 0.5rem;
+}
+
+.login-input .input-control{
+    display: flex;
+    flex-flow: column;
+    gap: 0.3rem;
```

Comment:
Suggestion : Better to use px for these small values, orelse when screen size decreased gap will become invisible